infra/metadata.rb

```diff
@@ -0,0 +1,6 @@
+name "reviewsite"
+version "0.1.0"
+
+depends 'rbenv', '1.7.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.